sklearn/decomposition/tests/test_pca.py

```diff
@@ -891,8 +891,16 @@ def check_array_api_get_precision(name, estimator, array_namespace, device, dtyp
 def test_pca_array_api_compliance(
     estimator, check, array_namespace, device, dtype_name
 ):
+    # Dask doesn't implement slogdet from the Array API
+    # which is used in score/score_samples
+    skip_methods = {"dask.array": ["score", "score_samples"]}
```

Comment:
Could you please add a TODO comment with a link to an upstream issue, either in `dask` or in `array-api-compat` to track the resolution of this problem?

Reply:
Actually, wouldn't it make sense to import `_array_api_skip_methods` from `sklearn.utils._array_api` instead of defining this dict inline here?

If so, lets move the comment there.